### PR TITLE
update bigdl version to 0.11.0

### PR DESCRIPTION
**Pull request by Le-Zheng**

### CODE REVIEW

File: zoo/pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
         <scala.macros.version>2.1.0</scala.macros.version>
         <scalatest.version>2.2.4</scalatest.version>
         <spark.version>2.4.3</spark.version>
-        <bigdl.version>0.10.0</bigdl.version>
+        <bigdl.version>0.11.0</bigdl.version>
```
Comment (qiyuangong) [inline zoo/pom.xml:30]: Need to update version in apps, docker and doc

Comment (hkvision) [reply zoo/pom.xml:30]: Also need to change python dependency and related doc.

Comment (Le-Zheng) [reply zoo/pom.xml:30]: Done

File: zoo/pom.xml [hunk 2]
```diff
@@ -27,7 +27,7 @@
         <scala.macros.version>2.1.0</scala.macros.version>
         <scalatest.version>2.2.4</scalatest.version>
         <spark.version>2.4.3</spark.version>
-        <bigdl.version>0.10.0</bigdl.version>
```
Comment (qiyuangong) [inline zoo/pom.xml:30]: 0.11 or 0.12?

Comment (qiyuangong) [inline zoo/pom.xml:30]: Does it mean we skipped 0.11?

**Pull request closed without merging**